lms/static/js/verify_student/views/make_payment_step_view.js

```diff
@@ -105,10 +105,20 @@ var edx = edx || {};
                 self._getProductText( templateContext.courseModeSlug, templateContext.upgrade )
             );
 
-            // create a button for each payment processor
-            _.each(processors.reverse(), function(processorName) {
-                $( 'div.payment-buttons' ).append( self._getPaymentButtonHtml(processorName) );
-            });
+            if (processors.length === 0) {
+                // No payment processors are enabled at the moment, so show an error message
+                this.errorModel.set({
+                    errorTitle: gettext('No available payment processors'),
```

Comment:
Thanks @srpearce, I've made those text changes. As Renzo said, `make_payment_step_view_spec.js` is a test file, so the same text will appear there.